<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnOffTestWidget</class>
 <widget class="QWidget" name="OnOffTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>板子号：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxMainDev"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>端口号：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxDev"/>
   </item>
   <item>
    <widget class="QLabel" name="labelPixmap">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/image/灯泡关.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonOn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>开</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonOff">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>关</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>pushButtonOff</zorder>
  <zorder>pushButtonOn</zorder>
  <zorder>labelPixmap</zorder>
  <zorder>pushButtonOff</zorder>
  <zorder>pushButtonOn</zorder>
  <zorder>comboBoxDev</zorder>
  <zorder>label</zorder>
  <zorder>comboBoxMainDev</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
